### PR TITLE
Drupal: Revert PR 2025 project pref XML changes

### PR DESCRIPTION
**Pull request by drshawnkwang**
Revert commits:
09b619bb97a950c024c1b53458dcb5d188c2bcdc
62b64347f70725af455acd6bec996ffd625d5e7b
3fead8d9a53f140ac3e7e60fcc22106fc6c1186d
c3d8d4dfcf3921a14c5ff683cd317d86779ba123

These four commits are associated with Pull Request (PR) #2025 , topic branch drshawnkwang:drupal_feature-projprefxml-apps.

Conflicts:
	drupal/sites/default/boinc/modules/boincwork/includes/boincwork.helpers.inc
Conflict was manually fixed.

See:
https://dev.gridrepublic.org/browse/DBOINC-87

### CODE REVIEW

**Comment by brevilo** [comment]
How did you revert the commits in question? Using `git revert` and the fixing the single conflict you stated?

**Comment by drshawnkwang** [comment]
I used git revert on the four commits one one line: `git revert abc def ghi jkl` and then fixed the conflict. I probably squashed it afterwards as well.

**Comment by brevilo** [comment]
For the record: I reverted the four commits in question (didn't find more) in the following order: c3d8d4d, 3fead8d, 62b6434, 09b619b

The final one (09b619b) lead to a conflict (via 70c867888c4 and 414fc988cc8) which I also resolved manually. I `diff`ed my version against Shawn's branch and the sole difference was an empty line I introduced (incorrectly).

tl;dr: I sign off on this PR.
